Makefile

```diff
@@ -0,0 +1,37 @@
+PROJ=peticodiac
+
+SRCS_DIR=src
+OBJS_DIR=obj
+HEADER_DIR=include
+
+SRCS=$(wildcard $(SRCS_DIR)/*.cpp)
```

Comment:
Nice. I didn't know about wildcard before: I would embed calls to find. Using this integrated feature of Make is better.